analyses/TX_cd_2020/03_sim_TX_cd_2020.R

```diff
@@ -0,0 +1,445 @@
+###############################################################################
+# Simulate plans for `TX_cd_2020`
+# © ALARM Project, February 2022
+###############################################################################
+set.seed(02138)
+library(sf)
+library(tidyverse)
+library(patchwork)
```

Comment:
these should be moved to `01_prep` in the block of stuff there.  If they're only used in diagnostic plots (e.g. `patchwork` then let's move the `library()` call to inside the first block of diagnostic plots